test/controllers/WebchatControllerSpec.scala

```diff
@@ -317,3 +333,5 @@ class WebchatControllerSpec
   }
 }
 
+
+
```

Comment:
```suggestion
```

Reply:
@sanjayshukla2205 there seems to be a couple of unnecessary new-lines. Could we remove them? (just accept the suggestion if you're happy with it)

Reply:
Done